process/project_proposals.md

```diff
@@ -11,23 +11,52 @@ This governance policy sets forth the proposal process for projects to be accept
 
-For moving levels to incubation or graduation, projects should plan on at least 5 months or more between the initial application and approval. Due to the increased community demands around KubeCon + CloudNativeCon, new public comment periods and public votes will not launch within 6 weeks of a KubeCon + CloudNativeCon. Project should take this into account when planning completion of their due diligence.
+When projects apply for moving levels, they do not move levels on a first-in first-out (FIFO) basis. Some projects join the Foundation when they are already far along in their journey and others join very early. They will approach maturity guide posts differently - at different times and in ways similar or different from others. We do our best to support projects moving levels, in some cases engaging with them frequently to ensure they are making progress on our initial findings but leaving the application open.  In other cases, we review the project and finding nothing disparaging, only needing to refresh the content of the previous due diligence, evaluate against the next level of criteria, and move it forward.
+
+For sandbox proposals, applications are reviewed in the order the TOC chooses, most commonly returning applications first and then from oldest application to newest every two months. The TOC may not have time to get through every application each meeting as different project applications carry varying considerations and discussion topics to ensure enough information is discovered and explored to make an informed decision. The up to date list can be found [here](https://sandbox.cncf.io/) and will be carried over from meeting to meeting if not every project is reviewed.
+
+For moving levels to incubation or graduation, projects should plan on _at least 5 months or more_ between the initial application and approval. 
+
+#### KubeCon+CloudNativeCon Freeze
+
+Due to the increased community demands around KubeCon + CloudNativeCon (KCCN), the scheduling and production of content, and reduced availability of individuals involved in moving levels, the TOC leverages a freeze for projects in process for moving levels. Even if a project is approved to move levels 3 weeks before this event, projects should _not_ expect to receive benefits beyond those afforded for the level they were previously at. For example, if a sandbox project is approved to move to incubation 3 weeks prior to the event, the project and the event staff will not have enough time to record, edit, and produce an incubating project update to have it included within the keynote stage reel.
+
+For each KubeCon + CloudNativeCon's (KCCN) standalone event — currently Europe and North America — the following freeze is applied:
```

Comment:
KCCN China is a thing again: https://www.lfasiallc.com/kubecon-cloudnativecon-open-source-summit-china/
```suggestion
For each KubeCon + CloudNativeCon's (KCCN) standalone event — currently Europe, China and North America — the following freeze is applied:
```

Reply:
So that's the tricky bit. KCCN China isn't a standalone event, while it is an event, it is run with OSS Summit: https://www.lfasiallc.com/kubecon-cloudnativecon-open-source-summit-china/

Additionally, obligations of the TOC members, projects, and other community members are not as heavy during KCCN China+OSS Summit. If those obligations picked up, and we were to have the freeze cover all three, we'd have several weeks where we'd not be able to make substantial progress on moving levels.  

This is going to be a problem moving forward (particularly if we create more standalone events that require new obligations or extend existing obligations) so i fully expect this to be revisited at a later date.

Does this make sense? We're trying to strike a good balance between everything without overburdening everyone or delaying projects moving levels significantly.

Reply:
Certainly, I guess I was not sure what "standalone event" meant so we might want to clarify that or add a footnote.

It's fine to ignore this though.

Thanks for explaining

Reply:
I've added additional clarity on this.